full-service/src/service/gift_code.rs

```diff
@@ -641,8 +641,7 @@ where
         transaction_builder.set_fee(Mob::MINIMUM_FEE)?;
 
         let num_blocks_in_ledger = self.ledger_db.num_blocks()?;
-        transaction_builder.set_tombstone_block(num_blocks_in_ledger + 50);
-
+        transaction_builder.set_tombstone_block(num_blocks_in_ledger + 10);
```

Comment:
I'm totally new to rust and FS, but it seems to me like this value of 10 here should come from the constant `DEFAULT_NEW_TX_BLOCK_ATTEMPTS` rather than be hard coded?

Reply:
Very happy to import it from the `transaction_builder` service. Is it best left as a public constant off of that service, or should we move the const to an external constants provider (a .env or similar)?

Reply:
It looks like there's the TODO for making this value configurable. Maybe once we implement that it will make sense to move this to config file, but until then I think it makes sense to import it from `transaction_builder.rs`.

Reply:
Maybe update the documentation for use that variable name instead of the number 10? that way if we tweak it again we don't have to update all the documentation as well?

Reply:
Currently I don't feel good about directing folks to look into the `transaction_builder` for this value when they're looking at the documentation. When we move this constant to a more unified location then I think we should update the docs to point to the constant at that point, but we should do that along with other constants that have been pulled in. Navigating docs should require minimal lift on the readers behalf.